_includes/header.html

```diff
@@ -3,7 +3,7 @@
.37573,-0.754681 -0.747773,-0.754681 h -1.349027 c -0.203632,0 -0.295442,-0.152446 -0.295442,-0.26013 v -0.26764 c 0,-0.168554 0.112703,-0.266286 0.309719,-0.266286 h 1.494044 c 0.182712,0 0.320087,-0.131579 0.320087,-0.305519 v -0.08013 c 0,-0.174617 -0.140066,-0.306257 -0.327903,-0.306257 H 17.908686 Z M 2.3438495,16.647915 v 2.896121 c 0,0.0908 0.071765,0.162295 0.1624483,0.162295 h 0.063117 c 0.090007,0 0.1640823,-0.07233 0.1632004,-0.162295 L 2.733,18.723 l 0.8067514,-5.1e-5 c 0.3770438,-2.4e-5 0.611473,-0.21544 0.611473,-0.561576 v -1.105587 c 0,-0.340873 -0.2405956,-0.561049 -0.611473,-0.560981 l -1.06952,1.95e-4 c -0.090167,1.6e-5 -0.1263819,0.07596 -0.1263819,0.152915 z m 4.7317457,-0.140188 c -0.075792,-0.0027 -0.1311917,0.0592 -0.1311917,0.140053 v 1.899644 c 0,0.08998 0.071119,0.162296 0.16275,0.162296 h 0.062819 c 0.090007,0 0.1633501,-0.07305 0.1633501,-0.162296 v -1.437247 c 0,-0.134677 0.1099709,-0.245244 0.2446487,-0.245244 h 0.5396409 c 0.1350831,0 0.2451002,0.110567 0.2451002,0.245244 v 1.437247 c 0,0.08998 0.071478,0.162296 0.1622985,0.162296 h 0.063414 c 0.090007,0 0.1631994,-0.07305 0.1631994,-0.162296 v -1.491941 c 0,-0.340883 -0.2396041,-0.56098 -0.6119234,-0.56098 -0.6280842,0 -0.7191073,0.02529 -1.0641057,0.01322 z m -1.8321651,8.4e-5 c -0.3135007,-1.4e-5 -0.547906,0.235805 -0.547906,0.525821 v 1.133226 c 0,0.293915 0.2386048,0.543253 0.5201047,0.543253 h 0.7620501 c 0.094193,0 0.1622975,-0.06535 0.1622975,-0.154037 v -0.04986 c 0,-0.08916 -0.068103,-0.153134 -0.1622975,-0.153134 H 5.3516291 c -0.1754318,0 -0.2680927,-0.08628 -0.2680927,-0.250209 v -0.33165 h 1.1294745 c 0.1816388,0 0.2889801,-0.110741 0.2889801,-0.298449 v -0.439406 c 0,-0.290006 -0.2458019,-0.525511 -0.5470042,-0.525524 z m -4.69492113,3.07e-4 c -0.30241703,1e-6 -0.5485071977581,0.235376 -0.5485071977581,0.525515 0,0.380598 0,0.761195 0,1.141793 0,0.294726 0.2381472377581,0.534829 0.5301734177581,0.534829 0.246202,0 0.49240401,0 0.73860601,0 0.2920262,0 0.5301745,-0.240103 0.5301745,-0.534829 v -1.141793 c 0,-0.290006 -0.2462256,-0.525514 -0.5485083,-0.525514 -0.2339795,0 -0.46795895,-2e-6 -0.70193843,-1e-6 z m 2.42951163,0.356899 h 0.5393403 c 0.1377814,0 0.2454008,0.107612 0.2454008,0.245254 v 0.997236 c 0,0.135344 -0.1104541,0.245398 -0.2454008,0.245398 H 2.9780206 c -0.135082,0 -0.2454007,-0.110054 -0.2454007,-0.245398 v -0.997236 c 0,-0.137786 0.1077538,-0.245254 0.2454007,-0.245254 z m -2.36609544,1.54e-4 c 0.19185251,0 0.38370503,0 0.57555754,0 0.1201031,0 0.2222583,0.102287 0.2222583,0.222262 0,0.347538 0,0.695076 0,1.042614 0,0.123073 -0.099456,0.223155 -0.2222583,0.223155 -0.19325499,0 -0.38650998,0 -0.57976497,0 -0.12307227,0 -0.21895167,-0.09788 -0.21895167,-0.223155 0,-0.347538 0,-0.937627 0,-1.042614 0,-0.122263 0.10035871,-0.222262 0.2231591,-0.222262 z m 5.28385294,0 c 0.1222628,0 0.2175994,0.09613 0.2175994,0.218507 V 17.43757 H 5.0835364 c 0,0 1.539e-4,-0.246767 1.539e-4,-0.353892 0,-0.122396 0.1119,-0.218507 0.2356619,-0.218507 z" />
```

Comment:
That change would have to be discussed with the board, and most importantly legal

Reply:
Reverted.